src/pudl/transform/eia.py

```diff
@@ -543,24 +544,15 @@ def harvesting(  # noqa: C901
     mcs = consistency["consistent_ratio"].mean()
     logger.info(f"Average consistency of static {entity} values is {mcs:.2%}")
 
-    if entity == "plants":
-        entity_df = _add_additional_epacems_plants(entity_df)
-        entity_df = _add_timezone(entity_df)
```

Comment:
I moved this plant-specific modification into the `harvested_entity_asset_factory`